src/terragraph-e2e/scripts/reload_topology_names.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sv restart stats_agent
+/usr/bin/fluent-bit restart
```

Comment:
This does not work whatsoever, you are looking for `/etc/init.d/fluent-bit restart`

Reply:
While executing "**/etc/init.d/fluent-bit restart**" facing below issue:

**no /usr/bin/fluent-bit found; none killed
cat: /var/run/fluent-bit/fluent-bit-output.conf: No such file or directory**

But while executing "**/usr/bin/fluent-bit  restart**" it's working as below:

**Fluent Bit v1.9.1
* Copyright (C) 2015-2021 The Fluent Bit Authors
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2022/11/10 06:56:02] [ info] [fluent bit] version=1.9.1, commit=67b144340b, pid=8547
[2022/11/10 06:56:02] [ info] [storage] version=1.1.6, type=memory-only, sync=normal, checksum=disabled, max_chunks_up=128
[2022/11/10 06:56:02] [ info] [cmetrics] version=0.3.0
[2022/11/10 06:56:02] [ info] [sp] stream processor started**

Reply:
Addressed the comments

While executing the **fluent-bit restart** it's working as below :

**I1115 12:29:29.565413  3724 ConfigApp.cpp:252] Changed or removed config values:
{"topologyInfo":{"nodeName":"my-node-12"}}
I1115 12:29:29.565675  3724 ConfigApp.cpp:509] Reload topology names...
ok: run: stats_agent: (pid 9255) 1s
Fluent Bit v1.9.1
* Copyright (C) 2015-2021 The Fluent Bit Authors
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2022/11/15 12:29:30] [ info] [fluent bit] version=1.9.1, commit=67b144340b, pid=9261
[2022/11/15 12:29:30] [ info] [storage] version=1.1.6, type=memory-only, sync=normal, checksum=disabled, max_chunks_up=128
[2022/11/15 12:29:30] [ info] [cmetrics] version=0.3.0
[2022/11/15 12:29:30] [ info] [sp] stream processor started**